content/en/account_management/billing/serverless.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Serverless billing
+kind: faq
+---
+
+## Overview
+
+Purchase serverless functions on [Datadog Pro and Enterprise plans][1]. Datadog bills based on the average of your accounts hourly functions across the month.  Pro and Enterprise plans include 40 custom metrics per billed function. Contact [Sales][2] or your [Customer Success Manager][3] for more information about adding Serverless to your account.
+
+## Serverless Functions
+
+Every hour, Datadog records the number of functions that execute one or more times on your Datadog account. At the end of the month, Datadog charges by calculating the average of of the hourly number of functions recorded.
+
+For Serverless pricing information, see the infrastructure section in [Datadogs pricing page][1].
+
+## Tracking Usage
+
+You can track the number of billable Serverless Functions in your account by checking the application [Usage Page][4]. You can see both the Month-To-Date summary, as well as usage over time.
+
+To control the number of functions Datadog is monitoring, filter out particular functions by sorting by tag with the API:
```

Comment:
we'll have this available in the UI by early next week, so customers will be able to filter by tag just like they do with EC2. we should incorporate that method here as the recommended solution

Reply:
oh cool! let me know when that is live and I can add it

Reply:
This is now live! https://cl.ly/bf32354bf132

Can we make this the recommended method of filtering Lambda functions? Might be worth keeping the API method for now in case people want that, but lower in the page